src/fastfield/bytes/reader.rs

```diff
@@ -31,36 +31,39 @@ impl BytesFastFieldReader {
         Ok(BytesFastFieldReader { idx_reader, values })
     }
 
-    fn range(&self, doc: DocId) -> (usize, usize) {
+    fn range(&self, doc: DocId) -> std::ops::Range<u64> {
```

Comment:
```suggestion
    fn range(&self, doc: DocId) -> Range<usize> {
```

Reply:
Why usize? Multivalues are u64

Reply:
The index you mean? They are consumed as usize and it is a private method here.

(Actually I think we should make all column indexes `u32` but that's another discussion.)

Reply:
for this PR you don't have to take the suggestions if you judge better as it is.

Reply:
The method is also used in the trait, which is u64. I also think we can change that, 4GB should be enough for a segment. For the time being I would be explicit with the type.